tests/HttpAdapterTest.php

```diff
@@ -1,22 +1,22 @@
 <?php
 
-namespace Http\Adapter\Tests;
+namespace Http\Adapter\Guzzle6\Tests;
 
 use GuzzleHttp\Client;
-use Http\Adapter\Guzzle6HttpAdapter;
+use Http\Adapter\Guzzle6\Client as Adapter;
```

Comment:
This is kind of weird. Use the `Client` class as an `Adapter` that actually wraps the Guzzle `Client`. I still prefer `Http\Adapter\Guzzle6\Client` over `Http\Adapter\Guzzle6\Adapter` though, as it makes clear that this class implements the `HttpClient` interface.